roles/test-utils/sv1-mining-device/Cargo.toml

```diff
@@ -13,6 +13,10 @@ keywords = ["stratum", "mining", "bitcoin", "protocol"]
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
+[lib]
+name = "sv1_mining_device"
```

Comment:
The `name` field can be removed since it's already defined in the `[package]` section. However, if we want to keep the original package name and use a different library name, including the `name` field in `[lib]` makes sense.

Reply:
this is true, however we have been following this same pattern with every roles

in case we decide to change this, IMO it should be a separate PR doing this in a consistent way across all roles

arguably cosmetic and therefore low prio IMHO